crates/sui-move/src/unit_test.rs

```diff
@@ -140,4 +149,6 @@ fn new_testing_object_and_natives_cost_runtime(ext: &mut NativeContextExtensions
     ext.add(NativesCostTable::from_protocol_config(
         &ProtocolConfig::get_for_max_version_UNSAFE(),
     ));
+
+    ext.add(store);
```

Comment:
Adding it to the extensions so that we can access it to add objects to it later when we allocate a receiving ticket.